### PR TITLE
Parameterize

### PR DESCRIPTION
**Pull request by thepwagner**
Not just a hardcoded scenario any more!

### CODE REVIEW

File: src/spacel/args.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        except HTTPError as e:
+            logger.warn('Unable to read manifest from %s: %s - %s', name,
+                        e.code,
+                        e.msg)
+            return None
+    elif url.scheme == 's3':
+        region, bucket, key = parse_s3(url)
+        s3 = boto3.resource('s3', region)
+        json_body = s3.Object(bucket, key).get()['Body'].read()
+    elif not url.scheme and url.path == '-':
```
Comment (DeviaVir) [inline src/spacel/args.py]: don't want to support local filesystem?

**Comment by DeviaVir** [comment]
+1; starts to look very interesting, would this already be able to drop-in replace our `_stack` in the current state?

File: src/spacel/args.py (hunk 1 above)
Comment (thepwagner) [inline src/spacel/args.py]: Good call; https://github.com/pebble/spacel-provision/issues/3

**Comment by thepwagner** [comment]
Re: replacement, I think so.

**Pull request merged**